### PR TITLE
Update Map Markers

### PR DESCRIPTION
**Pull request by tleraj**
The purpose of this PR is to close https://github.com/aqualinkorg/aqualink-app/issues/511.
Working example here: https://aqualink-app-programize.web.app/map

### CODE REVIEW

**Comment by tleraj** [comment]
@ericboucher Let me know if we want any changes on the marker sizes :slightly_smiling_face:

File: packages/website/src/routes/HomeMap/Map/Markers/index.tsx [hunk 1]
```diff
@@ -29,6 +35,48 @@ const buoyIcon = (iconUrl: string) =>
...
+) => {
+  switch (true) {
+    case hasSpotter && hasHobo:
+      return sensorIcon(true, selected, color);
+    case hasSpotter && !hasHobo:
+      return sensorIcon(true, selected, color);
+    case !hasSpotter && hasHobo:
+      return sensorIcon(false, selected, color);
+    case !hasSpotter && !hasHobo:
+      return buoyIcon(iconUrl);
```
Comment (ericboucher) [inline packages/website/src/routes/HomeMap/Map/Markers/index.tsx:74]: let's just reach the default, no?

File: packages/website/src/routes/HomeMap/Map/Markers/index.tsx [hunk 2]
```diff
@@ -29,6 +35,48 @@ const buoyIcon = (iconUrl: string) =>
     popupAnchor: [0, -28],
   });
 
+const sensorIcon = (isSpotter: boolean, selected: boolean, color: string) => {
```
Comment (ericboucher) [inline packages/website/src/routes/HomeMap/Map/Markers/index.tsx:38]: Maybe use an enum `spotter` | `hobo` instead of a boolean?

**Review by ericboucher — APPROVED** [review]
Minor comments, looks goods!

**Review by ericboucher — APPROVED** [review]

**Pull request merged**